src/main/resources/mustache-templates/search.html

```diff
@@ -2,29 +2,25 @@
 
 
 <div class="container vertical-space">
-	<h4>Patient Search</h4>
-	<small class="text-muted">Search the EHR system for a patient.
-		Click on the patient to see their labs and HPO terms.</small>
-	<form role="form" class="form" method="GET"
-		enctype="multipart/form-data" action="{{req.contextPath}}/search/">
+	<h3>Patient Search</h3>
+	<small class="text-muted">Search the EHR system for a patient.</small>
+	<form method="POST" action="{{req.contextPath}}/search">
+		{{#patientSearchForm}}
 		<div class="form-group row vertical-space">
-			<label for="first_name" class="col-sm-2 col-form-label">First
-				Name</label>
+			<label for="first_name" class="col-sm-2 col-form-label">First Name</label>
 			<div class="col-sm-4">
-				<input id="first_name" class="form-control" type="text"
-					name="first_name" />
+				<input class='form-control' type='text' name='firstName' value='{{#firstName}}{{.}}{{/firstName}}' />
```

Comment:
Super small thing, but I'd prefer if the quote style was consistent.

Reply:
Thanks, too much copying and pasting.